addons/delivery/static/src/js/location_selector/map/map.js

```diff
@@ -48,6 +48,9 @@ export class Map extends Component {
                 this.leafletMap = L.map(this.mapRef.el, {
                     zoom: 13,
                 });
+                this.leafletMap.attributionControl.setPrefix(
+                    '<a href="https://leafletjs.com" title="A JavaScript library for interactive maps">Leaflet</a>'
```

Comment:
Just out of curiosity, is there a guideline about mentioning leaflet somewhere? Shouldn't we include the Ukrainian flag like in their examples?

nitpicking: I think adding a pipe to separate the prefix from the copyright is clearer :smiley: 
```suggestion
                    '<a href="https://leafletjs.com" title="A JavaScript library for interactive maps">Leaflet</a> <span aria-hidden="true">|</span> '
```

Reply:
It just feels a bit out of place since we want Odoo to remain as politically neutral as possible